### PR TITLE
Version update in vpa deployments

### PR DESCRIPTION
**Pull request by mwielgus**
cc: @kgrygiel @tkulczynski

### CODE REVIEW

**Comment by tkulczynski** [comment]
I'm afraid 0.0.1 is also hardcoded in Makefiles and in hack/vpa-process-yaml.sh...

**Comment by mwielgus** [comment]
OK, i will update all places but we should eliminate them on the earliest convenience and have one file where registery and tag is defined.

File: vertical-pod-autoscaler/deploy/admission-controller-deployment.yaml [hunk 1]
```diff
@@ -13,19 +13,19 @@ spec:
       serviceAccountName: vpa-admission-controller
       containers:
       - name: admission-controller
-        image: k8s.gcr.io/vpa-admission-controller:v0.0.1
+        image: gcr.io/google_containers/vpa-admission-controller:v0.1.0
```
Comment (bskiba) [inline vertical-pod-autoscaler/deploy/admission-controller-deployment.yaml]: I think the current 'right way' is to use k8s.gcr.io?

**Comment by mwielgus** [comment]
Actually, makefiles don't need to be updated.

**Comment by mwielgus** [comment]
Updated.

File: vertical-pod-autoscaler/hack/vpa-process-yaml.sh [hunk 2]
```diff
@@ -31,8 +31,8 @@ if [ $# -eq 0 ]; then
   exit 1
 fi
 
-DEFAULT_REGISTRY="k8s.gcr.io"
-DEFAULT_TAG="v0.0.1"
+DEFAULT_REGISTRY="gcr.io/google_containers/vpa-updater"
```
Comment (kgrygiel) [inline vertical-pod-autoscaler/hack/vpa-process-yaml.sh]: Should be just "gcr.io/google_containers".

Comment (mwielgus) [reply vertical-pod-autoscaler/hack/vpa-process-yaml.sh]: Done.

**Pull request merged**